tests/contrib/pylons/test_pylons.py

```diff
@@ -32,6 +32,11 @@ def start_response_exception(self, status, headers):
         e.code = 'wrong formatted code'
         raise e
 
+    def start_response_string_code(self, status, headers):
+        e = Exception("Custom exception")
+        e.code = '512'
```

Comment:
I would keep both endpoints so you're testing that:
* with `'512'` it returns `512`
* with `'wrong status code'` it returns `500`